src/wp-admin/includes/class-wp-terms-list-table.php

```diff
@@ -735,7 +735,7 @@ public function inline_edit() {
 
 				<?php
 				wp_admin_notice(
-					'<p class="error"></p>',
+					'<p class="notice notice-error"></p>',
```

Comment:
Not a blocker but in 6.4 we introduce admin notice functions https://core.trac.wordpress.org/ticket/57791 can we use those here?

Reply:
This is not a correct usage of the notice-error class; those classes are already added to this admin notice by setting 'type' to 'error'; this class is inside the notice, and is probably expected by a JS selector to insert the notice when it is present. When changing 'error' classes, we only want to be changing them when they're styling the notice, not when they're used internally within the notice.